### DISCUSSION TITLE
JSON Items

### DISCUSSION THREAD

**Discussion by Kolhun** [nlohmann/json#4127] [Q&A] [answered]
Hello everyone, tell me there is a string, I translate it into JSON (json::parse). The question is that the keys I need are located in the "data" object how do I access the keys that are located in it? And work directly with it
![image](https://github.com/nlohmann/json/assets/36815965/97200e38-78f7-49c8-9839-19a8ddca997d)

**Comment by nlohmann** [comment, marked as answer]
See https://json.nlohmann.me/features/element_access/unchecked_access/
